package.json

```diff
@@ -33,5 +33,10 @@
     "gulp-ng-annotate": "^2.0.0",
     "gulp-using": "^0.1.1",
     "merge-stream": "^1.0.0"
+  },
+  "dependencies": {
+    "angular-animate": "^1.6.6",
+    "angular-sanitize": "^1.6.6",
+    "angular-ui-bootstrap": "^2.5.0"
```

Comment:
The front-end dependencies are handled by bower, there's no need to duplicate them in npm.

Reply:
Removed